tests/test_ast_symbol_extractor.py

```diff
@@ -298,6 +299,57 @@ def test_star_import():
     }
 
 
+def test_relative_import():
+    code = """
+    from . import core
+    from .core import ones
+    from ..core import twos
+    """
+    z = process_code_str(code)
+    assert z.symbols == {
+        "mm.core": {
+            "type": "relative-import",
+            "data": {"shadows": "core", "level": 1},
+        },
+        "mm.ones": {
+            "type": "relative-import",
+            "data": {"shadows": "core.ones", "level": 1},
+        },
+        "mm.twos": {
+            "type": "relative-import",
+            "data": {"shadows": "core.twos", "level": 2},
+        },
+        "mm": {
+            "type": "module",
+            "data": {},
+        },
+    }
+
+
+def test_relative_star_import():
+    code = """
+    from .core import *
+    from ..numeric import *
+    """
+    z = process_code_str(code)
+    expected_relative_imports = [
+        {
+            "symbol": "core",
+            "level": 1,
+            "module": "mm",
+        },
+        {
+            "symbol": "numeric",
+            "level": 2,
+            "module": "mm",
+        },
+    ]
+    assert (
+        sorted(z.symbols["relative-*"]["data"]["imports"], key=itemgetter("symbol"))
```

Comment:
not sure what symbol name to use here - I don't think it really matters since we will be looking up the relative imports by type anyway.

Not sure if we need the module where the symbol was imported, I think we might need it when we do the post processing of the relative imports in packages, but if we don't need it I will remove

Reply:
Yeah, I think it is ok to have the `relative-*` name. I guess you could have the name be something like `*<module_qual_name>` if you wanted better indexing, but it might not matter.

Reply:
yeah, once we start with the post processing we will get a feel for what the optimal model/naming conventions should be.